From e214b79d45cccdd0cfe839e54da2b3c82b6c6be4 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Thu, 31 Mar 2022 23:43:15 -0500
Subject: [PATCH 006/117] dt-bindings: display: Add D1 HDMI compatibles

Allwinner D1 contains a DesignWare HDMI controller with some changes in
platform integration, and a new HDMI PHY. Add their compatibles.

Signed-off-by: Samuel Holland <samuel@sholland.org>
---
 .../bindings/display/allwinner,sun8i-a83t-dw-hdmi.yaml           | 1 +
 .../bindings/display/allwinner,sun8i-a83t-hdmi-phy.yaml          | 1 +
 2 files changed, 2 insertions(+)

--- a/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-dw-hdmi.yaml
+++ b/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-dw-hdmi.yaml
@@ -29,6 +29,7 @@ properties:
           - enum:
               - allwinner,sun8i-h3-dw-hdmi
               - allwinner,sun8i-r40-dw-hdmi
+              - allwinner,sun20i-d1-dw-hdmi
               - allwinner,sun50i-a64-dw-hdmi
           - const: allwinner,sun8i-a83t-dw-hdmi
 
--- a/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-hdmi-phy.yaml
+++ b/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-hdmi-phy.yaml
@@ -19,6 +19,7 @@ properties:
       - allwinner,sun8i-a83t-hdmi-phy
       - allwinner,sun8i-h3-hdmi-phy
       - allwinner,sun8i-r40-hdmi-phy
+      - allwinner,sun20i-d1-hdmi-phy
       - allwinner,sun50i-a64-hdmi-phy
       - allwinner,sun50i-h6-hdmi-phy
 
